pkg/query-service/main.go

```diff
@@ -34,6 +36,10 @@ func main() {
 	logger := loggerMgr.Sugar()
 	version.PrintVersion()
 
+	go func() {
```

Comment:
starting a server should be moved to `server.Start()` method so that we can handle errors better. Also, handle the `error` object returned by function `http.ListenAndServe` method. 

We might consider later how to use structured code to handle `os.signal` and gracefully shut down query-service

Reply:
Done